modules/serverless-config-high-workloads-serving.adoc

```diff
@@ -0,0 +1,68 @@
+* The activator component also might need more memory, when it has to buffer more or bigger request payloads, so the memory requests and limits might need to be increased as well.
+* One activator pod can handle approximately 2500 requests per second before it starts to increase latency and, at some point, leads to errors.
+* One 3scale-kourier-gateway or istio-ingressgateway pod can also handle approximately 2500 requests per second before it starts to increase latency and, at some point, leads to errors.
+* Each of the data-plane components consumes up to 1 vCPU of CPU for handling 2500 requests per second. Note that this highly depends on the payload size and the response times of the Knative Service backend.
+
+[IMPORTANT]
+====
+Fast startup and fast response-times of your Knative Service user workloads are critical for good performance of the overall system. As Knative Serving components are buffering incoming requests when the Knative Service user backend is scaling-up or request concurrency has reached its capacity. If your Knative Service user workload introduces long startup or request-latency, it will either overload the `activator` component (when the CPU + memory configuration is too low) or lead to errors for the calling clients.
+====
+
+To fine-tune your installation, use the above findings combined with your own test results to configure the `KnativeServing` custom resource:
+
+
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 2 <1>
+  workloads:
+    - name: component-name <2>
+      replicas: 2 <2>
+      resources:
+        - container: container-name
+          requests:
+            cpu: <3>
+            memory: <3>
+          limits:
+            cpu: <3>
+            memory: <3>
+  podDisruptionBudgets: <4>
+    - name: name-of-pod-disruption-budget
+      minAvailable: 1
+----
+<1> Set this to at least 2, to make sure you have always at least two instances of every component running. You can also use `workloads` to override the replicas for certain components.
+<2> Use the `workloads` list to configure specific components. Use the `deployment` name of the component (like `activator`, `autoscaler`, `autoscaler-hpa`, `controller`, `webhook`, `net-kourier-controller`, `3scale-kourier-gateway`, `net-istio-controller`) and set the `replicas`.
+<3> Set the requested and limited CPU + memory according to at least the idle consumption while also taking the above findings and your own test results into consideration.
+<4> Adjust the `PodDistruptionBudgets` to a value lower or equal to the `replicas`, to avoid problems during node maintenance. The default `minAvailable` is set to `1`, so if you increase the desired replicas, make sure to also increase `minAvailable`.
```

Comment:
I think we should rather recommend just "lower to the replicas", not equal.  Setting it to equal would generate a PodDisruptionBudgetAtLimit alert, 

https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-controller-manager-operator/PodDisruptionBudgetAtLimit.md

> Standard workloads should have at least one pod more than is desired to support [API-initiated eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/). Workloads that are at the minimum disruption allowed level violate this and could block node drain. This is important for node maintenance and cluster upgrades.